src/Symfony/Component/Notifier/Bridge/TurboSms/TurboSmsTransportFactory.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\TurboSms;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Artem Henvald <genvaldartem@gmail.com>
+ */
+final class TurboSmsTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * @return TurboSmsTransport
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('turbosms' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'turbosms', $this->getSupportedSchemes());
+        }
+
+        $authToken = $this->getUser($dsn);
+        $from = $dsn->getRequiredOption('from');
```

Comment:
I would use the user for the "from" part and the password for the authtoken part.

Reply:
"from" can have a space, it can be a registered trade mark with space inside, e.g. "Walt Disney", "American Express". For now I use this SMS service for a local taxi service, its name is "**** taxi". So that's why it is moved to the option field in DSN instead of `user` part

Reply:
AFAIK you need to urlencode both of them, regardless of user or an option 🧐

Reply:
No problem. I can use `user` part of DSN instead of `from` option. But I looked into other bridges. There are a lot of DSNs which use `from` option https://symfony.com/doc/current/notifier.html#sms-channel 
For example `clickatell://ACCESS_TOKEN@default?from=FROM`
So I just did same as in other bridges to have the similar logic

Reply:
In case of `clickatell` it is really not needed and optional, so this in fine in this case.

However, i would prefer to use `user` and `pass` for this bridge

Reply:
ok, will change it

Reply:
Thank you 🚀

Reply:
Open for finishing this PR? 😃

Reply:
I will finish it, but a bit later, now have some problems, they take all time

Reply:
I think we can keep it like this, only one rebase and if the tests pass, lets merge it